UI/FacadeBase.pm

```diff
@@ -2118,6 +2118,24 @@ password.
 
 You may contact customer support by replying to this message.
 
+Thank you,
+vs_site_name(); Support
+EOF
+                    ],
+                    user_lockout_mail => [
+                        to => q{Mailbox(['Model.UserLoginForm', 'Email.email']);},
+                        subject => 'vs_site_name(); Account Locked',
+                        body => <<'EOF',
+We have locked your account due to a large number of failed login attempts. This is done to prevent
+potential attackers from being able to guess your password. To regain access to your account, follow
+link below or use the forgot password form.
```

Comment:
For a lockout, they must contact support.

Reply:
What's the reasoning for not allowing the user to unlock their account themselves?

Reply:
It makes sense to me, because that's what banks do. This is an extreme case, and we want to have some "back and forth" to verify that the person has access to the account (who is your club administrator, when did you last successfully login, etc.). I looked in NIST-63B, and there's nothing about what to do other than "Requiring the claimant to wait following a failed attempt for a period of time that increases as the account approaches its maximum allowance for consecutive failed attempts (e.g., 30 seconds up to an hour)."  That doesn't make much sense to me. The safest way to insert a significant delay is to contact support.